base/linalg/lapack.jl

```diff
@@ -94,6 +100,17 @@ function chkfinite(A::StridedMatrix)
     return true
 end
 
+# LAPACK version number
+function laver()
+    major = BlasInt[0]
+    minor = BlasInt[0]
+    patch = BlasInt[0]
+    ccall((@blasfunc(ilaver_), liblapack), Void,
```

Comment:
Just looks strange to be using what looks like, but isn't, a defined Julia object.